cg/cli/workflow/raredisease/base.py

```diff
@@ -39,3 +40,20 @@ def panel(context: CGConfig, case_id: str, dry_run: bool) -> None:
             click.echo(bed_line)
         return
     analysis_api.write_panel(case_id=case_id, content=bed_lines)
+
+
+@raredisease.command("managed-variants")
+@OPTION_DRY
+@ARGUMENT_CASE_ID
+@click.pass_obj
+def managed_variants(context: CGConfig, case_id: str, dry_run: bool) -> None:
+    """Write managed variants file exported from Scout."""
+
+    analysis_api: RarediseaseAnalysisAPI = context.meta_apis["analysis_api"]
```

Comment:
```suggestion
    analysis_api: RarediseaseAnalysisAPI = context.meta_apis["analysis_api"]
```
I see the "analysis_api" repeated so often throughout our code, does this not also warrant a constant?